R/check.R

```diff
@@ -68,18 +71,19 @@ check_group_date_unique <- function(obs) {
 #'
 #' @return NULL
 #'
+#' @importFrom cli cli_abort
 #' @family check
 check_module <- function(module) {
   if (!"data" %in% names(module)) {
-    stop(
-      "Must contain a list component specifying the data requirements for
-       further modelling as a list"
+    cli::cli_abort(
```

Comment:
as we aren't using package::function for the most part and `cli` functions are all prefixed with the package name do we really need to do so here?

Reply:
i'd be fine with not using `cli::cli_...` - I thought we had written down somewhere that we generally preferred the `package::function` syntax, but I think prefixed ones could be an exception (assuming we do even have that rule? not finding it).